docs/internal/investigations/dapr-sidecars-limitations.md

```diff
@@ -1,5 +1,4 @@
-# Remote Kyma Runtime configuration
-
+# Dapr sidecars limitations 
```

Comment:
That title is crazy ;) Maybe just name it like: Dapr reloading investigation, or something like that